src/tagger/imgurClient.js

```diff
@@ -0,0 +1,35 @@
+const imgur = require('imgur')
+
+const TaggerDatabaseManager = require('./TagDatabaseManager')
+
+class ImgurClient {
+  constructor () {
+    this.tdbm = new TaggerDatabaseManager()
```

Comment:
just use the instance from the tagger client, we don't need so many database managers